packages/frontend/src/components/Explorer/ExploreTree/TableTree/Tree/TreeSingleNode.tsx

```diff
@@ -110,6 +115,7 @@ const TreeSingleNode: FC<Props> = ({ node }) => {
                     {item.hidden ? (
                         <Tooltip
                             withArrow
+                            withinPortal
```

Comment:
I feel like those two should be a default props. thoughts?

Reply:
perhaps the withArrow @IrakliJani ! Don't know about withinPortal at first glance!